templates/circom/example-input.json

```
@@ -0,0 +1,5 @@
+{
+    "X": 1,
+    "Y": 1,
+    "isEqual": 1
```

Circom (unlike Gnark) does not include output signals as the input to the circuit.  For this circuit, the `public` field sent back from the API for the proof detail includes "isEqual" as the last entry
```suggestion
    "Y": 1
```